update.cf

```diff
@@ -31,6 +29,64 @@ body common control
 }
 
 #############################################################################
+
+bundle agent mpf_update
+# @brief Masterfiles Policy Framework Update Policy
+#
+# @description These policies are related to ensuring the environment is
+# configured properly before a policy run. This includes publishing policy with
+# the masterfiles-stage tooling, making sure that $(sys.inputdir) is updated
+# from its upstream masterfiles, updating CFEngine packages, and ensuring the
+# necessary CFEngine components are running,
+{
+  vars:
+      "sorted_update_bundlesequence_end"
+        meta => { "defvar" },
+        slist => sort( @(def.update_bundlesequence_end), "lex" );
```

Comment:
`@(list)` is erratically supported until 3.9 (where functions started taking lists and data containers and parsing var refs more consistently). `list` is supported for sure.